bazarr.py

```diff
@@ -89,12 +128,22 @@ def daemon():
             except:
                 print('Unable to delete restart file.')
             else:
-                start_bazarr()
+                start_bazarr(daemonStatus)
+    
+    
+    daemonStatus = DaemonStatus()
     
+    def shutdown(signal_no=signal.SIGINT):
+        # indicates that everything should stop
+        daemonStatus.stop()
+        daemonStatus.interrupt(signal_no)
+        # emulate a Ctrl C command on itself (bypasses the signal thing but, then, emulates the "Ctrl+C break")
+        os.kill(os.getpid(), signal_no)
     
-    start_bazarr()
+    signal.signal(signal.SIGTERM, lambda signal_no, frame: shutdown(signal.SIGINT))
```

Comment:
So here the signal we send to the children processes is actually a "nice" interruption. There is an open question about whether we should actually send a more brutal one...

Reply:
Is it possible to first send a "nice" signal then, if it fails to exit, for ce kill the process?

Reply:
Working on it.
I'll introduce a timeout of 2 minutes will that be sufficient ? (I think it should from what I've observed)

Reply:
2 minutes should be way enough!